script/ListBidEssay.s.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+
+import {OneSevenTwoNineEssay} from "../src/1729Essay.sol";
+import {Solenv} from "solenv/Solenv.sol";
+
+contract ListBidEssayScript is Script {
+
+    function run() public {
+
+        // Load config from .env
+        Solenv.config();
+        address TOKEN_ADDRESS = vm.envAddress("TOKEN_ADDRESS");
+        address MULTISIG_ADDRESS = vm.envAddress("MULTISIG_ADDRESS");
+        address AUCTION_HOUSE_ADDRESS = vm.envAddress("AUCTION_HOUSE_ADDRESS");
+        address MODULE_MANAGER_ADDRESS = vm.envAddress("MODULE_MANAGER_ADDRESS");
+        address TRANSFER_HELPER_ADDRESS = vm.envAddress("TRANSFER_HELPER_ADDRESS");
+
+        // Parameters
+        uint256 TOKEN_ID = vm.envUint("TOKEN_ID");
+        address AUTHOR_ADDRESS = vm.envAddress("AUTHOR_ADDRESS");
+        uint256 AUCTION_DURATION = vm.envUint("AUCTION_DURATION");
+        uint256 AUCTION_RESERVE_PRICE = vm.envUint("AUCTION_RESERVE_PRICE");
+        uint256 BID_AMOUNT = vm.envUint("BID_AMOUNT");
+
+        // set up auctionhouse
+        ReserveAuctionCoreETH auctionHouse = ReserveAuctionCoreETH(AUCTION_HOUSE_ADDRESS);
+
+        // get already deployed Essay NFT contract
+        OneSevenTwoNineEssay nft = OneSevenTwoNineEssay(TOKEN_ADDRESS);
+
+        vm.startBroadcast();
+        // authorize zora contracts
+        nft.setApprovalForAll(TRANSFER_HELPER_ADDRESS, true);
+        ModuleManager(MODULE_MANAGER_ADDRESS).setApprovalForModule(
+            AUCTION_HOUSE_ADDRESS, true
+        );
+
+        vm.stopBroadcast();
+
+        vm.startBroadcast();
+        // list essay
+
+        auctionHouse.createAuction(
+            TOKEN_ADDRESS,
+            TOKEN_ID,
+            AUCTION_DURATION,
+            AUCTION_RESERVE_PRICE,
+            AUTHOR_ADDRESS,
+            block.timestamp
+        );
+
+        // verify listing
+        (
+            address seller,
+            uint256 reservePrice,
+            address fundsRecipient,
+            ,
+            ,
+            uint256 duration,
+            ,
+        ) = auctionHouse.auctionForNFT(address(nft), 1);
+        require(seller == MULTISIG_ADDRESS);
+        require(reservePrice == AUCTION_RESERVE_PRICE);
+        require(fundsRecipient == AUTHOR_ADDRESS);
+        require(duration == AUCTION_DURATION);
+        vm.stopBroadcast();
```

Comment:
Be consistent with whitespace and `stopBroadcast()` — either do like here and no blank line above, or do like the other places and leave a blank line above.